test/index.spec.js

```diff
@@ -79,16 +79,10 @@ function testInlineSourceWebpackPlugin(
       ).toBeTruthy();
     }
     if (expectResults && expectResults.length) {
-      const content = stats.compilation.assets[outputFile].source();
-      expect(
-        expectResults.every(result => {
-          if (result instanceof RegExp) {
-            return result.test(content);
-          } else {
-            return content.indexOf(result) > -1;
-          }
-        })
-      ).toBeTruthy();
+      const content = fs.readFileSync(path.join(__dirname, `../dist/${outputFile}`)).toString();
+      expectResults.forEach(result => {
+        expect(content).toMatch(result);
+      });
```

Comment:
The compilation source no longer contains the content after emit.

https://webpack.js.org/blog/2020-10-10-webpack-5-release/#sizeonlysource-after-emit

> webpack now replaces the Sources in Compilation.assets with SizeOnlySource variants to reduce memory usage.